baseband/mark4/base.py

```diff
@@ -520,4 +524,13 @@ def _make_frame(self, frame_index):
     :class:`~baseband.mark4.base.Mark4FileWriter` (binary), or
     :class:`~baseband.mark4.base.Mark4StreamReader` or
     :class:`~baseband.mark4.base.Mark4StreamWriter` (stream)
+
+Notes
+-----
+One can also pass in a list, tuple, or subclass of
```

Comment:
Maybe add "Although it is not generally expected to be useful for Mark 4, like for the other formats, ..." - or do we have examples of multiple Mark 4 files?

This then also makes it a bit more logical that the mode is not tested.

Reply:
I don't think I've ever seen that.  Adding the line.